WrightTools/data/_data.py

```diff
@@ -897,10 +897,9 @@ def create_variable(
     def downscale(self, tup, name=None, parent=None) -> "Data":
         """Down sample the data array using local averaging.
 
-        See `skimage.transform.downscale_local_mean`__ for more info.
+        See `scipy.signal.decimate`__ for more info.
```

Comment:
Left over from first rewrite, if we move forward with this one, needs updating. 

Also applies to internal function name